File: exercicios_beecrownd/1175.c
#include <stdio.h>


int main(){

    int A[20] = {0};
    int aux = 0;

    for(int i=0; i<20; i++){
        scanf("%d", &A[i]);
      
    }

    for(int i=0; i<10; i++){
   

        aux = A[i];
        A[i] = A[19-i];
        A[19-i] = aux;
      
     
        }
       
    for(int i=0; i<20; i++){
         printf("N[%d] = %d\n", i, A[i]);
    }




    return 0;
}

File: exercicios_beecrownd/1175.py
vetor = []


for i in range(20):
    a = int(input())
    vetor.append(a)
    
for i in range(10):
    aux = vetor[i]
    vetor[i] = vetor[19-i]
    vetor[19-i] = aux

for i in range(20):
    print('N[%d] ='%i,vetor[i])
    

